src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -1799,6 +1800,30 @@ public IEnumerable<CollectionStats> GetCollections(DocumentsOperationContext con
             }
         }
 
+        public CollectionDetails GetCollectionDetails(KeyValuePair<string, long> collection, DocumentsOperationContext context)
+        {
+            CollectionDetails collectionDetails = new CollectionDetails() { Name = collection.Key, CountOfDocuments = collection.Value, Size = new Client.Util.Size() };
+            CollectionName collectionName = GetCollection(collection.Key, throwIfDoesNotExist: false);
+
+            if (collectionName != null)
+            {
+                Table collectionTable = context.Transaction.InnerTransaction.OpenTable(DocsSchema,
+                    collectionName.GetTableName(CollectionTableType.Documents));
```

Comment:
Also need to take into account:
* Conflicts, Revisions and Tombstones.

Reply:
Isn't the DocSchema in this context always "Documents"? How can I take your mentioned types into account? Could you give me an advise, please?

Reply:
Yes, `DocsSchema` is for documents. 
But a collection is also composed of other items, listed above. 
Each one of them has its own schema. See `RevisionStorage` for example.

`Revisions`, in particular, may be a significant portion of a collection size

Reply:
Thanks for the advice.
Conflicts seems to be complicated, because they are not directly associated with a collection.
There is no CollectionTableType for conflicts. I tried to use the values of ConflictsStorage.ConflictsSchema and ConflictsStorage.ConflictsSlice to get the table + report, but the result is not referred to a collection. Or am I wrong in that?

Reply:
Yes, you're right. Conflicts don't have per collection table. Only the following ones:

https://github.com/ravendb/ravendb/blob/5ee2e9602d646c4f3e50633854b3eb1bbf6fb2cd/src/Raven.Server/Documents/CollectionName.cs#L64-L70

`CollectionTableType.Counters` is legacy value, currently we use `CounterGroups`